### PR TITLE
new mapping for interop api

### PR DESCRIPTION
**Pull request by anthdm**

### CODE REVIEW

File: compiler/syscall.go [hunk 1]
```diff
@@ -1,69 +1,72 @@
...
-	"GetType":         "Neo.Transaction.GetType",
-	"GetAttributes":   "Neo.Transaction.GetAttributes",
-	"GetInputs":       "Neo.Transaction.GetInputs",
-	"GetOutputs":      "Neo.Transaction.GetOutputs",
-	"GetReferences":   "Neo.Transaction.GetReferences",
-	"GetUnspentCoins": "Neo.Transaction.GetUnspentCoins",
-	"GetScript":       "Neo.InvocationTransaction.GetScript",
-
-	// TODO: Add the rest of the interop APIS
+var syscallss = map[string]map[string]string{
```
Comment (jeroenptrs) [inline compiler/syscall.go]: Syscalls has an extra 's' here

File: compiler/syscall.go [hunk 2]
```diff
@@ -1,69 +1,72 @@
...
+	},
+	"asset": {
+		"GetAssetID":   "Neo.Asset.GetAssetID",
+		"GetAssetType": "Neo.Asset.GetAssetType",
+		"GetAmount":    "Neo.Asset.GetAmount",
+	},
+	"contract": {
+		"GetScript": "Neo.Contract.GetScript",
+		"IsPayable": "Neo.Contract.IsPayable",
+	},
```
Comment (jeroenptrs) [inline compiler/syscall.go]:
```
"GetStorageContext": "Neo.Contract.GetStorageContext",
"Create": "Neo.Contract.Create",
"Migrate": "Neo.Contract.Migrate",
"Destroy": "Neo.Contract.Destroy"
```

are missing

Comment (jeroenptrs) [reply compiler/syscall.go]: Also, where does `IsPayable` come from?

File: compiler/syscall.go (hunk 1 above)
Comment (anthdm) [reply compiler/syscall.go]: This is coming in my next PR, will land in about 1 hour :)

File: compiler/syscall.go (hunk 2 above)
Comment (anthdm) [reply compiler/syscall.go]: I found IsPayable in the NEO BOA interop. Is `IsPayable` not a thing?. Will add the missing interop APIS.

**Pull request merged**